### ISSUE TITLE
[Enhancement] Specify if the front or rear camera is opened in CapturePhotoAsync

### ISSUE THREAD

**Issue by ssbssb-cmd** [xamarin/Essentials#1699]
## Summary
Please provide a brief summary of your proposal. Two to three sentences is best here.

## API Changes

Include a list of all API changes, additions, subtractions as would be required by your proposal. These APIs should be considered placeholders, so the naming is not as important as getting the concepts correct. If possible you should include some "example" code of usage of your new API.  You should also provide details of the level of availability for the feature on each of the supported platforms.

e.g.

In order to facilitate the new Shiny Button api, a bool is added to the Button class. This is done as a bool because it is simpler to data bind and other reasons...

    var button = new Button ();
    button.MakeShiny = true; // new API

The MakeShiny API works even if the button is already visible.

## Intended Use Case
Provide a detailed example of where your proposal would be used and for what purpose.

**Comment by 98gzi** [comment]
Also agree, it's annoying to keep switching the view when you are attaching multiple captured images.

**Comment by sshere** [comment]
Can everyone please clarify, Is everyone looking for the ability to:

1) See if the camera is already open on a specific device and detect which camera is being used of the two.
2) The ability to see which camera is active within the application currently and where the image was captured from?
3) The ability to change the source (Front or Rear) of the Camera?

**Comment by benjamingranados** [comment]
Yeah! This feature is needed... maybe could be set in the MediaPickerOptions, some sort of:

```
var photo = await MediaPicker.CapturePhotoAsync(new MediaPickerOptions
{
    Title ="Please take a selfie",
    FromRear = false // or true as a default
});
```
